sys-cluster/teleport/teleport-3.2.4.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2019 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```

Comment:
EAPI-7 should work here. Can you test it?

You can update the sign-off status by editing the Github issue title and appending `[please reassign]` there.
(@gyakovlev)